packages/primitives/src/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { Image } from 'theme-ui'
+import { SxStyleProp } from '@kodiak-ui/core'
+import { Box } from '../Box'
+
+export type AvatarProps = {
+  imageSrc?: string
```

Comment:
This should be able to just be `src` instead of `imageSrc`. We should utilize the default HTML attributes for an image instead of adding a custom prop that does the same thing.